feat:improve documentation (#119)
industrial-asset-hub/asset-link-sdk: docs/bootstrap.md

[7/8] company_url (https://www.device-builder.local): https://www.device-builder.local
[8/8] year (2024): 2024
```
![Bootstrapping Steps](https://github.com/industrial-asset-hub/asset-link-sdk/tree/main/docs/images/bootstrap-steps.gif)

> **Note: As a recommendation, Asset Link should be named in a way that clearly includes the phrase “Asset Link”. This ensures a consistent and recognizable naming convention for created Asset Link. Example: Custom Asset Link, Default Asset Link.**

Reviewer: ```suggestion
**Note: As a recommendation,  the Asset Link naming should include the identifier “Asset Link”. This ensures a consistent and recognisable naming convention for created Asset Link. For example: Custom Asset Link, Default Asset Link.**
```

> The command above binds the Asset Link to a publicly accessible IP address on your host.
> Please ensure that the port is protected from external access.

## Handler Implementation and Interfaces

> To implement your own discovery logic, open the **handler/handler.go** file. This Go module contains the core implementations for Asset Link functionality, including device discovery and configuration handling. You can customize functions such as `Discover`, `GetSupportedOptions`, and `GetSupportedFilters` to fit your specific requirements for actual device discovery and to retrieve information about the discovery parameters that are supported by the specific Asset Link .

Reviewer: ```suggestion
To implement your own discovery logic, open the **handler/handler.go** file. This Go module contains the core implementations for Asset Link functionality, including device discovery and configuration handling. You can customize functions such as `Discover`, `GetSupportedOptions`, and `GetSupportedFilters` to fit your specific requirements for actual device discovery and to retrieve information about the discovery parameters that are supported by the specific Asset Link.
```
